content/post/release-of-1.4.0.md

```diff
@@ -0,0 +1,189 @@
+---
+date: "2018-01-25T12:00:00+00:00"
+author: "thehowl"
+title: "Gitea 1.4 is released"
+tags: ["release"]
+draft: true
+---
+
+Hello! Yet another release cycle is over; this time around we merged [X pull requests](https://github.com/go-gitea/gitea/pulls?utf8=%E2%9C%93&q=is%3Apr+milestone%3A1.4.0+is%3Amerged) - we made sure to deliver as much as we could in this new Gitea version, and we're sure you're going to like it.
+
+Starting from this release, we'll also make sure to walk you through the most important changes we made - instead of just dumping the PRs closed in this release and their category. And boy have we got a lot of stuff to talk about.
+
+<!--more-->
+
+## Reactions ([#2856](https://github.com/go-gitea/gitea/pull/2856))
+
+![Screenshot of Reactions](/demos/2856/1.png)
+
+![Screenshot of Reactions](/demos/2856/2.png)
+
+Yes. Yes! YES!!! Gitea now has reactions on pull requests and issues. Now hopefully your collaborators will shut up with their "+1" comments on the issue and will instead just press the 👍 button. Oh yeah 😎.
```

Comment:
Maybe not telling collaborators to "shut up" 😂 How about `Maybe now your inbox will not be filled with "+1" emails` instead. Still tounge-in-cheek, while not offending your collaborators 😉